<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SejourTermine</class>
 <widget class="QDialog" name="SejourTermine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#numchts_txt
{
color:black;
background:transparent;

}#typelogts_txt
{
color:black;
background:transparent;

}#prenomts_txt
{
color:black;
background:transparent;

}
#dats_txt
{
color:black;
background:transparent;

}
#fraisejour_txt
{
color:black;
background:transparent;

}
#dpts_txt
{
color:black;
background:transparent;

}
#cints_txt
{
color:black;
background:transparent;

}
#nomts_txt
{
color:black;
background:transparent;

}
#nbrsejour_txt
{
color:black;
background:transparent;

}
#label_numero
{
color:black;
background:transparent;

}
QDialog#SejourTermine
{
		background-image: url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
#frame
{

	background:rgb(255, 255, 255);
border-radius:15px;
}


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }


#label
{
color:black;
background:transparent;

}
#label_2
{
color:black;
background:transparent;

}
#label_3
{
color:black;
background:transparent;

}
#label_4
{
color:black;
background:transparent;

}
#label_5
{
color:black;
background:transparent;

}

#label_6
{
color:black;
background:transparent;

}#label_7
{
color:black;
background:transparent;

}#label_8
{
color:black;
background:transparent;

}#label_9
{
color:black;
background:transparent;

}#label_10
{
color:black;
background:transparent;

}#label_11
{
color:black;
background:transparent;

}#label_12
{
color:black;
background:transparent;

}#label_13
{
color:black;
background:transparent;

}#label_14
{
color:black;
background:transparent;

}
#numero
{
color:black;
background:transparent;

}
QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover
{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>551</width>
     <height>551</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="prenomts_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="fraisejour_txt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date départ :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="cints_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="numchts_txt">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>390</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="typelogts_txt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Numéro de chambre :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nbrsejour_txt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN :</string>
    </property>
   </widget>
   <widget class="QLabel" name="dpts_txt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dats_txt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frais de séjour :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type logement :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date arrivée :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nomts_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de séjour :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>86</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/projetimages/projetimages/e.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff0000;&quot;&gt; Hotel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmerts_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminer séjour</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
